tools/scylla-sstable.cc

```diff
@@ -671,7 +671,7 @@ class writetime_histogram_collecting_consumer : public sstable_consumer {
                 }
             });
         } else {
-            throw std::runtime_error(fmt::format("Cannot collect timestamp of cell (column {} of uknown type {})", cdef.name_as_text(), cdef.type->name()));
```

Comment:
> fix typo;
> Signed-off-by: Yuao Ma <c8ef@outlook.com>

This repository expects better changelog messages, use `git log` to get a sense of how they are formatted.

Reply:
Also, please set the patch author to your name.